types/node-telegram-bot-api/index.d.ts

```diff
@@ -422,6 +429,9 @@ declare namespace TelegramBot {
         permissions?: ChatPermissions | undefined;
         can_set_sticker_set?: boolean | undefined;
         sticker_set_name?: string | undefined;
+        has_private_forwards?: boolean | undefined;
+        has_protected_content?: boolean | undefined;
+        is_automatic_forward?: boolean | undefined;
```

Comment:
I think this prop should't be on the Chat interface, just on the Message interface. Can you share more information?

Reply:
Sorry for the mistake ;) 
Besides, I added  `slow_mode_delay`, `message_auto_delete_time` and `linked_chat_id` to the Chat interface. API document is [here](https://core.telegram.org/bots/api#chat). Please review them, thanks :)